addons/web/static/src/js/fields/basic_fields.js

```diff
@@ -2154,8 +2154,11 @@ var PriorityWidget = AbstractField.extend({
         this.$el.empty();
         this.empty_value = this.field.selection[0][0];
         _.each(this.field.selection.slice(1), function (choice, index) {
-            self.$el.append(self._renderStar('<a href="#">', index_value >= index+1, index+1, choice[1]));
+            self.$el.append(self._renderStar('<a>', index_value >= index + 1, index + 1, choice[1]));
```

Comment:
what about rendering a `<span>` tag instead of a link when the field is readonly?

Reply:
@aab-odoo 👍